src/components/Download/Download.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useEffect } from "react";
+import { Table, NavLink, Input, Row, Col,Alert } from "reactstrap";
+import { IconContext } from "react-icons";
+import { FaDownload } from "react-icons/fa";
+import "./Download.css";
+
+export default () => {
+  const [Platform, setPlatform] = useState("");
```

Comment:
can we make this `Platform` to `platform` in order to follow camelCase as you did with other two variables.
What do you think , would it  be nice?

Reply:
@endurance21 done